docs/cloud/core-concepts.md

```diff
@@ -82,58 +80,54 @@ Examples of fields:
 
 ## Namespace
 
-Namespace is a group of streams in a source or destination. Common use cases for namespaces are enforcing permissions, segregating test and production data, and general data organization.
-
-A schema in a relational database system is an example of a namespace.
+Namespace is a method of grouping streams in a source or destination. Namespaces are used to generally organize data, segregate tests and production data, and enforce permissions. In a relational database system, this is known as a schema.
 
-In a source, the namespace is the location from where the data is replicated to the destination.
+In a source, the namespace is the location from where the data is replicated to the destination. In a destination, the namespace is the location where the replicated data is stored in the destination. 
 
-In a destination, the namespace is the location where the replicated data is stored in the destination. Airbyte supports the following configuration options for destination namespaces:
+Airbyte supports the following configuration options for a connection:
 
-<table>
-  <tr>
-   <td><strong>Configuration</strong>
+   <table>
+   <tr>
+   <td><strong>Destination Namespace</strong>
    </td>
    <td><strong>Description</strong>
    </td>
-  </tr>
-  <tr>
-   <td>Mirror source structure
+   </tr>
+   <tr>
+   <td>Destination default
    </td>
-   <td>Some sources (for example, databases) provide namespace information for a stream. If a source provides the namespace information, the destination will reproduce the same namespace when this configuration is set. For sources or streams where the source namespace is not known, the behavior will default to the "Destination default" option.
+   <td>All streams will be replicated to the single default namespace defined by the Destination. For more details, see<a href="https://docs.airbyte.com/understanding-airbyte/namespaces#--destination-connector-settings"> ​​Destination Connector Settings</a>
```

Comment:
```suggestion
   <td>All streams will be replicated to the single default namespace defined by the Destination. For more details, see<a href="/understanding-airbyte/namespaces#--destination-connector-settings"> ​​Destination Connector Settings</a>
```

It is usually better to use relative links so that they still work when developing locally or in staging sites (vercel previews)

Reply:
Updated!